FlowCryptUI/Nodes/ComposeButtonNode.swift

```diff
@@ -15,7 +15,8 @@ public final class ComposeButtonNode: ASButtonNode {
         super.init()
         onTap = action
         backgroundColor = .main
-        setTitle("+", with: UIFont.boldSystemFont(ofSize: 30), with: .white, for: .normal)
+        accessibilityIdentifier = "composeMessageButton"
```

Comment:
We now have a set format for accessibility identifiers: always start with `aid` (means accessibility id) and separate with dashes. So here let's do `aid-compose-message-button`